### PR TITLE
emulated_hue note for sleep cycle and sleep as android

### PR DESCRIPTION
**Pull request by orrpan**
**Description:**
Note describing how to use sleep cycle and sleep as android in emulated hue using full state view

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#26650

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/emulated_hue.markdown [hunk 1]
```diff
@@ -33,6 +33,12 @@ Both Google Home and Alexa use the device they were initially set up with for co
 
 </div>
 
+<div class='note'>
+
+<a href="https://www.sleepcycle.com" target="_blank">Sleep Cycle</a> and <a href="https://sleep.urbandroid.org" target="_blank"> Sleep as Android</a>: smart alarm clock app can use emulated_hue to turn on and off enitites. Sleep Cycle only has it implemented in the iOS app, see <a href="https://support.sleepcycle.com/hc/en-us/articles/207670385-Does-Sleep-Cycle-integrates-with-Phillips-Hue-" target="_blank">Sleep Cycle support</a>. The app requires the same configuration as Google Home and does not work if type is defined as alexa in configuration.
```
Comment (frenck) [inline source/_components/emulated_hue.markdown]: Please use markdown in this paragraph instead of HTML.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @orrpan, I've reviewed your PR and left a comment. Could you please take a look? Thanks! 👍

**Comment by frenck** [comment]
Looking at the result, I feel like there is an overload on notes:

image

Can we reduce these notes into a list? For example, in combining them in a paragraph?

**Comment by frenck** [comment]
A small and friendly notification.

Are you planning on moving forward on this PR? 
I didn't see any activity recently, while there are still things to do. I hope you consider picking it up soon! 👍

**Comment by orrpan** [comment]
Yes, but I’m unsure if your other comments is relevant for adding a new note. I agree that alot of notes is not the cleanest way to go, but should that not be done in another pr? (Which focuses on the docs, not a new functionality)

I could try introducing more changes to the layout, but is it in scope?

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Rebased PR to resolve merge conflicts.

**Review by frenck — APPROVED** [review]
Thanks for your comments @orrpan!

I agree it is something that needs to be addressed, but maybe a bit out of scope.

Thanks for your contribution! 🎉

**Comment by orrpan** [comment]
@frenck, I see that this was merged, but the referred PR is not (see description). So this functionality does not work jet

**Pull request merged**